engine/gamecontroller.py

```diff
@@ -1,8 +1,10 @@
 from engine.board import Board
-from engine.deck import Deck
-from engine.player import Player
+from engine.color import Color
+from engine.deck import (Deck, WildDeck)
```

Comment:
Oh, good call. I misunderstood that in the guidelines in `[elements of style](https://github.com/amontalenti/elements-of-python-style)`.